packages/data-service/src/connect-mongo-client.ts

```diff
@@ -90,7 +90,6 @@ export function prepareOIDCOptions(
       matchingAllowedHosts(connectionOptions);
   }
 
-  // @ts-expect-error Will go away on @types/node update
```

Comment:
I don't know why this affected me first? How does it not affect main?

Reply:
I think the second comment line here was also part of the expect-error description 🙂

Reply:
oops. fixed.